inlong-sort/sort-flink/sort-flink-v1.15/sort-connectors/redis/src/main/java/org/apache/inlong/sort/redis/source/RedisRowDataLookupFunction.java

```diff
@@ -107,6 +120,10 @@ public void eval(Object... keys) {
                         throw new UnsupportedOperationException(
                                 String.format("Unsupported for redisCommand: %s", redisCommand));
                 }
+                if (sourceMetricData != null) {
+                    sourceMetricData.outputMetricsWithEstimate(rowData, System.currentTimeMillis());
```

Comment:
no need to get current time here

see  
private long getCurrentOrProvidedTime(long dataTime) {
        return dataTime == 0 ? System.currentTimeMillis() : dataTime;
    }

Reply:
fixed it.